Doc/library/random.rst

```diff
@@ -303,6 +297,15 @@ be found in any statistics text.
 Alternative Generator
 ---------------------
 
+.. class:: Random([seed])
+
+    Class to be subclassed if you want to use a different basic generator of your
```

Comment:
Style nit: Please follow the existing style in `Doc/library/random.rst` and use 3-space indentation.

Reply:
Other than berker's nits, this looks fine and is ready to apply.  I added backport tags as well.

Reply:
Oops, yes, nice catch. done, and vimrc updated accordingly to not do that again.